modules/archetypes/lib/policy_set_definitions/policy_set_definition_es_enforce_guardrails_keyvault.tmpl.json

```diff
@@ -134,7 +134,7 @@
           "displayName": "The minimum days before expiry",
           "description": "Enter the days before expiration of the certificate when you want to trigger the policy action. For example, to trigger a policy action 90 days before the certificate's expiration, enter '90'."
```

Comment:
if the default is 10 we should probs update the description to match

Reply:
well it's written as an example but I updated it